### PR TITLE
Sniffer in python

### PR DESCRIPTION
**Pull request by pritamp17**
# Description

the script that can sniff HTTP pageg

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings

### CODE REVIEW

File: SNIFFER/readme.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# this  script is created in python
```
Comment (atarax665) [inline SNIFFER/readme.md:1]: I guess you can come up with a better README. Have a look at READMEs of other contributed projects to get some idea.

File: SNIFFER/sniffer.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import scapy.all as scapy
+from scapy.layers import http  # pip install scapy_http
+
+# from scapy_http import http
+
+def get_url(packet):
+    return packet[http.HTTPRequest].Host + packet[http.HTTPRequest].Path         #for detecting urls
```
Comment (atarax665) [inline SNIFFER/sniffer.py:7]: Comment should be above the code, not beside it.

**Review by atarax665 — COMMENTED** [review]
Please format your code according to PEP8 Guidelines. You can use [this](http://pep8online.com/).

File: SNIFFER/sniffer.py (hunk 2 above)
Comment (pritamp17) [reply SNIFFER/sniffer.py]: sir, I have made changes, do I have to make any other  changes?

**Review by avinashkranjan — APPROVED** [review]

**Pull request merged**